dummyserver/certs/server.ipv6addr.crt

```diff
@@ -1,16 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
I think the tests would be nice to have, good reference for doing similar stuff in the future at least.